CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 Changelog
 =========
 
+[1.3.0] - 2024-04-02
+--------------------
+
+### New Features
+
+- feat: Add options for rate limit interval and burst (#64)
+
+  Enhancement:
+  Add two new variables, allowing the rate limit to be configured using the role.
+  
+  Reason:
+  IHAC which needs these settings to be changed from their default values, and they would like to use RHEL System Roles as much as possible.
```

Comment:
I think lines 10 to 15 should be removed here as it doesn't sound right for a change log entry.

When I filled out the pull request template I wasn't aware that the body would be included in the change log, so now it comes out a bit weird (especially the "Reason" part).

Reply:
Well, we have automation that just automatically builds the changelog entries from the PR.
I suppose we could require two additional doc fragments when submitting a PR - one for the PR body with the details and the reason, and one for the changelog snippet, but that seems like too much overhead.
I agree in this case that it doesn't really add anything to the changelog

Reply:
@ephracis please modify PR description in #64 so it has a text suitable for a changelog entry?

Reply:
@spetrosi done! :)